test/functional/wallet_multisig_descriptor_psbt.py

```diff
@@ -96,9 +97,9 @@ def run_test(self):
         self.log.info("Check that every participant's multisig generates the same addresses...")
         for _ in range(10):  # we check that the first 10 generated addresses are the same for all participant's multisigs
             receive_addresses = [multisig.getnewaddress() for multisig in participants["multisigs"]]
-            all(address == receive_addresses[0] for address in receive_addresses)
+            assert_true(all(address == receive_addresses[0] for address in receive_addresses))
             change_addresses = [multisig.getrawchangeaddress() for multisig in participants["multisigs"]]
-            all(address == change_addresses[0] for address in change_addresses)
+            assert_true(all(address == change_addresses[0] for address in change_addresses))
```

Comment:
Good find!
In other cases we seem to simply compare with the expected boolean value, e.g. https://github.com/bitcoin/bitcoin/blob/master/test/functional/feature_addrman.py#L89 - is that an option here?